contracts/ClaimIssuer.sol

```diff
@@ -9,11 +10,11 @@ contract ClaimIssuer is IClaimIssuer, Identity {
     mapping (bytes => bool) public revokedClaims;
     mapping (bytes32 => address) public identityAddresses;
 
-    constructor(address _owner) public {
-        set(_owner);
+    function setClaimIssuer(address _owner) public {
+        postConstructor(_owner);
```

Comment:
What does `postConstructor` do? Not clear at all, we can find a better name that wouldn't conflict with other contract methods.

Reply:
Maybe simply "setOwner"?

Reply:
Yes, I agree but there is no ownership mechanism on this contract, maybe another keyword?

Reply:
Keyholder maybe?

Reply:
`setManager` then, if it's for a management key?

Reply:
yes ok go for it